Python/chapter01/1.6 - String Compression/lomeli_1.6.py

```diff
@@ -0,0 +1,29 @@
+# Implement a method to perform basic string compression using the counts
+# of repeated characters.  If the "compressed" string would not become smaller
+# than the original string, your method should return the original stringself.
+# You can assume that the string has only uppercase and lowercase letters.
+
+def stringCompress(st):
+    result = []
+    cur = st[0]
+    count = 0
+    for i in range(len(st)):
```

Comment:
given that you don't use the index (by itself) for anything, it's better to do:

```python
for c in st:
```

if you find yourself needing both the index and the character, you can use

```python
for i, c in enumerate(st):
```